src/listen.js

```diff
@@ -152,6 +152,23 @@ module.exports = function(defaultFuncs, api, ctx) {
                 })(0)
                 break;
               }
+              
+              if (v.delta.class == "ClientPayload") {
+                var clientPayload = utils.decodeClientPayload(v.delta.payload);
+                if (clientPayload && clientPayload.deltas) {
+                  for (var i in clientPayload.deltas) {
+                    var delta = clientPayload.deltas[i];
+                    if (delta.deltaMessageReaction) {
```

Comment:
Sorry, just one more thing. Can you explicitly create and return a new object?  This will help if facebook changes the format of the object they send us, I believe.
Why do you delete `delta.threadKey` and `delta.action`?